src/OctoshiftCLI.Tests/GithubApiTests.cs

```diff
@@ -138,6 +138,69 @@ public async Task CreateTeam_Returns_Created_Team_Id()
             result.Should().Be(teamId);
         }
 
+        [Fact]
+        public async Task GetTeams_Returns_All_Teams()
+        {
+            // Arrange
+            const string org = "ORG";
+
+            var url = $"https://api.github.com/orgs/{org}/teams";
+
+            const string team1 = "TEAM_1";
+            const string team2 = "TEAM_2";
+            var responsePage1 = $@"
+            [
+                {{
+                    ""name"": ""{team1}"",
+                    ""id"": 1
+                }},
+                {{
+                    ""name"": ""{team2}"", 
+                    ""id"": 2
+                }}
+            ]";
+
+            const string team3 = "TEAM_3";
+            const string team4 = "TEAM_4";
+            var responsePage2 = $@"
+            [
+                {{
+                    ""name"": ""{team3}"",
+                    ""id"": 3
+                }},
+                {{
+                    ""name"": ""{team4}"", 
+                    ""id"": 4
+                }}
+            ]";
+
+            async IAsyncEnumerable<JToken> GetAllPages()
```

Comment:
This may be a question for @ArinGhazarian but I'm expecting/hoping there is a way to write simpler tests without having to mock out paging/JSON in situations like this.

I don't think this test needs to test our paging implementation (we have separate unit tests for that), I would hope there is a way to simplify this test so it has the mock just return a simple list of strings (team names) instead of pages of JSON.

Reply:
Oh yeah, we most certainly can right simpler tests now. At the time I was writing my test for `GetRepos` I wasn't using `System.Linq.Async` library (to keep things simple), but now that we're using it we can simply convert any `IEnumerable` To `IAsyncEnumerable` with `ToAsyncEnumerable()` extension method like this:

```c#
var asyncEnumerable = new[] { item1, item2 }.ToAsyncEnumerable();
```

so that entire local method can go away and replaced by a `ToAsyncEnumerable` call on any `IEnumerable` 😃